lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -182,6 +182,30 @@ def get_cpu_facts(self, collected_facts=None):
             return cpu_facts
 
         cpu_facts['processor'] = []
+        cmd = self.module.get_bin_path('lscpu')
+        # cmd = cmd + " | egrep '^(Socket|Thread|Core|CPU|Model name)'"
+        rc, lscpu_out, err = self.module.run_command(cmd)
+        if lscpu_out:
+            for line in lscpu_out.splitlines():
+                data = line.split(' ')
```

Comment:
Rather than splitting on spaces, split on the colon, then clean up the spaces:
```suggestion
                data = line.split(':', 1)
                data[1] = data[1].lstrip()
```

Reply:
fixed